assistant/assistant.go

```diff
@@ -96,11 +97,9 @@ type response struct {
 	Data   interface{}
 }
 
-// Run starts the assistant listening to the server and handles and interrupts or closed connections
-func (assistant *Assistant) Run(ctx context.Context) error {
+func (assistant *Assistant) initWebsocketConnection(ctx context.Context) error {
```

Comment:
We probably want this loop of retrying to occur infinitely. So like it tries to connect, if it fails it waits 5 seconds then tries again, if it fails it waits another 5 seconds then tries again etc.. 

1. Attempts to connect to websocket, if it succeeds returns the connection and sets it to the assitants.
2. If it doesn't succeed, log error, sleep for x amount of seconds, then recall the function.

Reply:
Infinite loop in initWebsocketConnection already does this. Not sure if the infinite connecting part should actually be in the func or handled by funcs who call initWebsocketConnection.

Reply:
This part was already implemented early on, I just moved it to a separate function so that if the connection is dropped somehow after the initial connection is established, I could call this same retry loop (i.e. if the server drops the connection as opposed to the initial connection attempt when Telegraf starts up).